pvr.hts/resources/language/resource.language.en_gb/strings.po

```diff
@@ -50,10 +50,28 @@ msgctxt "#30007"
 msgid "Response timeout in seconds"
 msgstr ""
 
-#empty strings from id 30008 to 30099
+msgctxt "#30008"
+msgid "Advanced settings"
+msgstr ""
+
+#empty strings from id 30009 to 30049
+
+msgctxt "#30050"
+msgid "Auto recordings"
+msgstr ""
+
+msgctxt "#30051"
+msgid "Use approximate start time instead of fixed start time"
```

Comment:
This is better, imo.

However, for the seeing as a list of choices we need a name for the seeing as well as names for the two values.

How about:

"Start time window calculation" for the setting's name and "Start + End time" (strict approach) and "Start +- < the name of the other new setting we should imo rename too >" (relaxed approach).

Should we rename the other new setting to "Fixed start time window span (+- mins)"?